salt/modules/zk_concurrency.py

```diff
@@ -276,6 +277,8 @@ def unlock(path,
 
 def party_members(path,
                   zk_hosts,
+                  min_nodes,
+                  blocking=False
                   ):
     '''
     Get the List of identifiers in a particular party
```

Comment:
Can you add some documentation for your new arguments to explain what they're for? Also please be sure to include a version added directive for each new argument like this:
```
min_nodes

    .. versionadded:: Carbon

    Description of what min_nodes does.
```